### PR TITLE
📝 (Storage/UI): Added header and footer to the Storage Upload page

### PR DESCRIPTION
**Pull request by GO-Shoderu**
Added header and footer to the Storage Upload page

### CODE REVIEW

**Review by lindzik — DISMISSED** [review]
Tests ran successfully. Naming conventions check out. Approve merging.

**Comment by arneschreuder** [comment]
This PR is more than just a few simple fixes. I will let it slide, but please only small fixes to the release branch.

File: libs/client/student-profile/feature/src/lib/bio/bio.component.html [hunk 1]
```diff
@@ -1,10 +1,13 @@
-<mat-card *ngIf="bio">
+<mat-card>
     <mat-card-title color="primary">  BIO (Adding tags will make it easier for employers to find you) </mat-card-title>
     <mat-card-content>
-        <p> {{bio.bio}}</p>
+        <p> Growing up, I've always had an interest in computer science and found myself excelling in various fields of IT...</p>
```
Comment (LarisaBothaTuks) [inline libs/client/student-profile/feature/src/lib/bio/bio.component.html:4]: why did you add this bio directly to the html?

File: libs/client/student-profile/feature/src/lib/bio/bio.component.html [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
         <mat-card-actions>
             <mat-chip-list aria-label="Tags">
-                <mat-chip *ngFor="let tag of bio.tags">{{tag}}</mat-chip>
+                <!-- <mat-chip *ngFor="let tag of bio.tags">{{tag}}</mat-chip> -->
```
Comment (LarisaBothaTuks) [inline libs/client/student-profile/feature/src/lib/bio/bio.component.html:7]: more hard coded mock data?

File: libs/client/student-profile/feature/src/lib/user-info/user-info.component.html [hunk 3]
```diff
@@ -2,14 +2,14 @@
   <h2>Student Details</h2>
   <mat-divider class="heading"></mat-divider>
```
Comment (LarisaBothaTuks) [inline libs/client/student-profile/feature/src/lib/user-info/user-info.component.html:4]: these mock info should be added dynamically???

**Review by LarisaBothaTuks — APPROVED** [review]
I am confused on why we are adding mock data directly to the html that is not right please see if you can add it dynamically later on before the demo.

**Comment by GO-Shoderu** [comment]
All these changes doesn't belong to our feature ... I only made changes to one file today ... and I figure our last pr from last night wasn't merged automatically to the release ... perhaps we did the last pr after the merge to release was done yesterday night ... 

so in total changes or addition were made to 3 files for adding headers ad footers

**Review by lindzik — APPROVED** [review]
Changes made and files should be up to date with the release branch. Naming conventions followed. Approve merging
